apps/services/license-api/src/app/modules/license/license.service.ts

```diff
@@ -46,8 +41,46 @@ export class LicenseService {
       : new InternalServerErrorException([details ?? 'Unknown error'])
   }
 
+  private async getClientByLicenseId(
+    licenseId: LicenseId,
+  ): Promise<BaseLicenseUpdateClient> {
+    const type = mapLicenseIdToLicenseType(licenseId)
+
+    if (!type) {
+      this.logger.error(`Invalid license type ${type}`)
```

Comment:
```suggestion
      this.logger.error(`Invalid license type`, { category: LOG_CATEGORY, type })
```

https://www.notion.so/Logging-Discipline-42bdeac698c8482d9cfab163a83b6009